src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -110,24 +86,30 @@ def truncate_value(self, value: str) -> str:
 
     def set(self, key: str, value: Any) -> None:
         self.set_options.append((key, value))
+        self.options_changed = True
 
     def unset(self, key: str) -> None:
         self.unset_options.append(key)
+        self.options_changed = True
 
     def update(self, key: str, db_value: Any, value: Any) -> None:
         self.updated_options.append((key, db_value, value))
+        self.options_changed = True
 
     def channel_update(self, key: str) -> None:
         self.channel_updated_options.append(key)
```

Comment:
some have expressed interest in adding channel update to the slack message, this would require a change on the webhook side as well but can be addressed in another pr